exampleCourse/questions/demo/addHTMLtable/question.html

```diff
@@ -0,0 +1,34 @@
+
+
+<pl-question-panel>
+
+  <div class="card my-2">
+      <div class="card-header">
+          Example Table 1:
+      </div>
+      <div class="card-body">
+          {{{params.mytable}}}
+      </div>
+ </div>
+
+ <div class="card my-2">
+     <div class="card-header">
+         Example Table 2: adding table properties
+     </div>
+     <div class="card-body">
+         <table style="width:30%" border="black">
```

Comment:
What does this look like if we use the bootstrap table themes? 

```
<table class="table table-sm table-bordered">
```

This is probably fine how it is but I prefer how the bootstrap tables look over the normal html ones.

Reply:
I am okay either way. I was trying to have something very simple. Feel free to make changes for style, but keeping the simplicity.

Reply:
You're right that this should be a simple example.  I think it is fine as-is.